src/oc_erchef/apps/chef_index/src/chef_index_batch.erl

```diff
@@ -201,21 +246,33 @@ handle_cast({stats_update, TotalDocs, {AvgQueueLatency,AvgSuccessLatency}, Resp}
                                           total_docs_queued = TQ,
                                           total_docs_success = TS
                                          }) ->
-    State1 = State#chef_idx_batch_state{total_docs_queued = TQ + TotalDocs,
-                                        avg_queue_latency = ((AvgQueueLatency*TotalDocs)+(OQL*TQ))/(TQ+TotalDocs)
+
+    {message_queue_len, MailboxSize} = process_info(self(), message_queue_len),
+    prometheus_gauge:set(chef_index_batch_mailbox_size, MailboxSize),
```

Comment:
Would it be useful to add gauges for AvgQueueLatencyUpdated and AvgSuccessLatencyUpdated

Reply:
Because we have histograms for both of these I think we trivially get the average (we get sum and count as part of the histogram).  I'd like to eventually retire this older stats stuff so I'm trying to keep them separate.